README.md

```diff
@@ -47,3 +67,78 @@ To run this on AWS Lambda, see
 This repo uses semver, and its public interface (in order to determine what
 changes would break backwards-compatibility) is defined as the functions
 exported by `./index.js`. 
+
+## Checking PHP versions
+
+Each of the commands for scanning one or more repos also accepts an optional URL
```

Comment:
The scripts fail if there is no csv url, but here it says "optional". Is this inconsistent or am I reading it incorrectly?

Reply:
Oops. Apparently I never tested them without the CSV URL after adding that feature.

Reply:
Hmm, I'm confused. "It works on my machine." 😁 

Ohhh... I bet you're refering to the "PHP EOL" scripts. You're correct that they require a CSV URL. They just aren't listed in the README yet (as you observed elsewhere), which is why the README still says the CSV URL is optional... because it is for all of the documented calls.